### ISSUE TITLE
Why can not resume queue item from single downloads

### ISSUE THREAD

**Issue by AmirHosseinKarimi** [persepolisdm/persepolis#761]
Refer to this part of code, Why user can not resume an item from Single or All downloads that already exist/moved to an queue?

https://github.com/persepolisdm/persepolis/blob/ee8c78110623f0a91a1f5d9ae93464b4849358c7/persepolis/scripts/mainwindow.py#L2704-L2709

**Comment by alireza-amirsamimi** [comment]
Because that download item is belong to queue. The queue must be stopped or ...

**Comment by AmirHosseinKarimi** [comment]
There is not any way to resume item that belong a queue? It's technically restrictions?
Perspolise use aria2 to download files?! If it's true, why we can not resume the download anyway?

**Comment by alireza-amirsamimi** [comment]
User can stop or resume the queue.

**Comment by AmirHosseinKarimi** [comment]
Right, But if user need to resume only one item from queue? First should send item to single downloads and then resume the download?
Its not better if user have an ability to resume download of every single items in queue without moving them to single downloads list?

Its not an limitation from application? Or its technical limitation from aria2 or other part of third party applications?

**Comment by AmirHosseinKarimi** [comment]
@alireza-amirsamimi Hi, Any update?
